tests/mock_crust_tests.rs

```diff
@@ -40,6 +40,10 @@
 extern crate itertools;
 #[macro_use]
 extern crate log;
+#[cfg_attr(feature="clippy", allow(useless_attribute))]
+#[allow(unused_extern_crates)]
+#[macro_use]
+extern crate maidsafe_utilities;
```

Comment:
Shouldn't it be `#[cfg(test)]` or `#cfg[cfg_attr(feature="use-mock-crust")]` then?